Prevent the tests from exiting when they encounter an error.

This change will not be sent upstream.

--- libsndfile-1.0.28/tests/test_wrapper.sh.in.orig	2017-04-10 10:18:14.892226937 +0000
+++ libsndfile-1.0.28/tests/test_wrapper.sh.in	2017-04-10 10:18:29.399953955 +0000
@@ -61,7 +61,7 @@
 	fi
 
 # Force exit on errors.
-set -e
+#set -e
 
 # Generic-tests
 uname -a
